includes/compatibility/class-wcpdf-compatibility-third-party-plugins.php

```diff
@@ -192,6 +195,45 @@ public function add_wpc_product_bundles_classes ( $classes, $document_type, $ord
 		return $classes;
 	}
 
+	/**
+	 * YITH WooCommerce Product Bundles compatibility
+	 *
+	 * @param string $classes CSS classes for item row (tr)
+	 * @param string $document_type PDF Document type
+	 * @param object $order WC_Order order
+	 * @param int $item_id WooCommerce Item ID
+	 */
+	public function add_yith_product_bundles_classes( string $classes, string $document_type, object $order, int $item_id = 0 ): string {
+		if ( ! class_exists( 'YITH_WCPB' ) ) {
+			return $classes;
+		}
+
+		$item_id = ! empty( $item_id ) ? $item_id : $this->get_item_id_from_classes( $classes );
+
+		if ( empty( $item_id ) ) {
+			return $classes;
+		}
+
+		$product_id = (int) wc_get_order_item_meta( $item_id, '_product_id', true );
```

Comment:
Maybe you could instantiate the item and get the product using the proper method?